templates/pages/setup/crontask/crontask.html.twig

```diff
@@ -0,0 +1,177 @@
+   {% set params  = params ?? [] %}
+   {% set target       = params['target'] ?? item.getFormURL() %}
+   {% set withtemplate = params['withtemplate'] ?? '' %}
+   {% set item_type = item.getType() %}
+   {% set field_options = {} %}
+
+   <div class="card-body d-flex flex-wrap">
+      <div class="col-12 col-xxl-{{ item_has_pictures ? '9' : '12' }} flex-column">
+         <div class="d-flex flex-row flex-wrap flex-xl-nowrap">
+            <div class="row flex-row align-items-start flex-grow-1">
+               <div class="row flex-row">
+                  {% block form_fields %}
+                     {% set name = item.fields['name'] %}
+                     {% if plugin_info %}
+                        {% set name = __('%1$s - %2$s')|format(plugin_info['plugin'], name) %}
+                     {% endif %}
+                     {% set name_field %}
+                        <span class="fw-bold">{{ name }}</span>
+                     {% endset %}
+                     {{ fields.htmlField('name', name_field, __('Name')) }}
+
+                     {{ fields.textareaField(
+                        'comment',
+                        item.fields['comment'],
+                        _n('Comment', 'Comments', get_plural_number()),
+                        field_options
+                     ) }}
+
+                     {{ fields.htmlField('description', item.getDescription(item.getID), __('Description')) }}
+
+                     {{ fields.dropdownFrequency('frequency', item.fields['frequency'], __('Run frequency')) }}
+
+                     {# Status #}
+                     {{ fields.dropdownArrayField('state', item.fields['state'], {
+                        (constant('CronTask::STATE_DISABLE')): __('Disabled'),
+                        (constant('CronTask::STATE_WAITING')): __('Scheduled')
+                     }, __('Status')) }}
+
+                     {# Run mode #}
+                     {{ fields.dropdownArrayField('mode', item.fields['mode'], {
+                        (constant('CronTask::MODE_INTERNAL')): __('GLPI'),
+                        (constant('CronTask::MODE_EXTERNAL')): __('CLI')
+                     }, __('Run mode')) }}
+
+                     {# Run period #}
+                     {% set run_period_field %}
+                        <div class="d-flex">
+                           {{ fields.dropdownNumberField('hourmin', item.fields['hourmin'], null, {
+                              min: 0,
+                              max: 24,
+                              no_label: true
+                           }) }}
+                           <span class="mx-2 text-nowrap">-></span>
+                           {{ fields.dropdownNumberField('hourmax', item.fields['hourmax'], null, {
+                              min: 0,
+                              max: 24,
+                              no_label: true
+                           }) }}
+                        </div>
+                     {% endset %}
```

Comment:
This part renders packed too much with the field on the right.
I think we should remove col-12 attribute to avoid overextend of the fields